src/components/SourceSelect/SourceSelectDropdown.tsx

```diff
@@ -53,6 +55,11 @@ export function SourceSelectDropdown({
       }
     >
       <DropdownList className="ix-dropdown-list">
+        {!allSources.length ? (
+          <Paragraph style={{ paddingLeft: 5 }}>
```

Comment:
I don't recall if we ever took a hard stance on inline styling and when it can/should be used.